.circleci/config.yml

```diff
@@ -71,7 +71,6 @@ build_machine_environment: &build_machine_environment
 
     # These, mostly overlapping, flags ensure that CircleCI is as pretty as
     # possible for a non-interactive environment.  See also: --headless.
-    EMACS: t
```

Comment:
With `METEOR_HEADLESS` enabled, we're not really gaining anything by using this setting in CircleCI.